### PR TITLE
Remove companies, add co-chair

### PR DESCRIPTION
**Pull request by keith**
We've decided that the company model isn't as important as we originally
thought.

### CODE REVIEW

File: CONTRIBUTING [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 Mike Nakhimovich @digitalbuddha - Until 1/2025
-Justin Newitter @justinnewitter (LinkedIn) - Until 2/2023
-Oscar Bonilla @ob (LinkedIn, Secondary) - Until 2/2023
-Ty Smith @tyvsmith (Uber) - Until 2/2023
-Rudro Samanta @rudro (Uber, Secondary) - Until 2/2023
-Valera Zakharov @valeraz (Slack) - Until 2/2023
-Jonathan Schear @jschear (Slack, Secondary) - Until 2/2023
+Justin Newitter @justinnewitter - Until 2/2023
+Oscar Bonilla @ob - Until 2/2023
+Ty Smith @tyvsmith - Co-Chair - Until 2/2023
```
Comment (sberrevoets) [inline CONTRIBUTING]: Move up to the top?

**Review by sberrevoets — APPROVED** [review]
I suppose we should update these dates too at some point

**Review by ob — APPROVED** [review]

**Review by tyvsmith — APPROVED** [review]
LGTM

**Review by tyvsmith — CHANGES_REQUESTED** [review]
Actually, on second thought, we should remove the dates here. Some have already passed.

**Comment by keith** [comment]
I figured we should keep them as a reference until we've decided what to replace them with

**Review by tyvsmith — APPROVED** [review]

**Pull request merged**